pkg/payload/payload_test.go

```diff
@@ -149,39 +152,83 @@ func Test_Exclude(t *testing.T) {
 			isExcluded: true,
 		},
 		{
+			name:        "profile-selection-works",
```

Comment:
nit: no need to dash these by hand; we can just write phrases like `"profile selection works"` and `testing` will slug them for us.

Reply:
> nit: no need to dash these by hand; we can just write phrases like "profile selection works" and testing will slug them for us.

I'll defer to my reviewer, but that prevents a straight copy/find/paste flow in the code base.

Reply:
fixed in fda77fe; thanks :)

And fair enough on copy-paste-ness, but the existing CVO pattern is unslugged values.  I'm fine with a larger pivot in this direction if we want to shift conventions...